frontend/lib/map/preview/accepting_store_preview_card.dart

```diff
@@ -1,53 +1,48 @@
+import 'package:ehrenamtskarte/store_widgets/accepting_store_summary.dart';
 import 'package:flutter/material.dart';
 
 import '../../widgets/error_message.dart';
-import '../detail/detail_view.dart';
-import 'accepting_store_preview_card_content.dart';
 import 'models.dart';
 
 class AcceptingStorePreviewCard extends StatelessWidget {
   final bool isLoading;
   final Function refetch;
-  final /*?*/ AcceptingStoreSummary acceptingStore;
+  final /*?*/ AcceptingStoreSummaryModel acceptingStore;
```

Comment:
What does `/*?*/` mean?

Reply:
This is to mark an optional type (which is used once the null-safety migration is started)